pom.xml

```diff
@@ -26,7 +26,7 @@
   </parent>
   <groupId>org.apache.spark</groupId>
   <artifactId>spark-parent_2.11</artifactId>
-  <version>2.3.5-TDP-0.1.0-SNAPSHOT</version>
+  <version>2.3.4-1.0</version>
```

Comment:
TDP was based upon Spark 2.3.5-SNAPSHOT instead of a released 2.3.4 (https://github.com/TOSIT-IO/spark-fork/commit/8b7e61ed66e6e5279b2a57edddae03978e95e42f#diff-346f143033cd930581fe2f0e9e07532546867e2999c98cf990aa1109369f4fffL24-R24)

I think it's fair to use 2.3.4-1.0 in this case